src/pages/Feed.js

```diff
@@ -1,40 +1,43 @@
-// import AddPostForm from "../components/posts/AddPostForm.js";
 import PostList from "../components/posts/PostList.js";
 import { useState, useEffect } from "react";
 
-function FeedPage() {
+function FeedPage(props) {
+  const refreshPost = props.refreshPost;
   const [isLoading, setIsLoading] = useState(true);
   const [loadedPosts, setLoadedPosts] = useState([]);
 
-  useEffect(()=>{
+  useEffect(() => {
+    props.setRefreshPost(false);
     setIsLoading(true);
-    fetch("https://friendface-react-e85cf-default-rtdb.firebaseio.com/posts.json")
+    fetch(
+      "https://friendface-react-e85cf-default-rtdb.firebaseio.com/posts.json"
+      )
       .then((response) => {
         return response.json();
       })
       .then((data) => {
-        const posts = []
-
-        for(const key in data){
+        const posts = [];
+        
+        for (const key in data) {
           const post = {
             id: key,
-            ...data[key]
+            ...data[key],
           };
-
-          posts.push(post)
+          
+          posts.push(post);
         }
         setIsLoading(false);
         setLoadedPosts(posts);
       });
-  }, [])
-
+      // eslint-disable-next-line
+    }, [refreshPost]);
```

Comment:
So as a follow up with `useEffect`s this will only ever be re-called if the values inside this array is changed. Beforehand when you had nothing in here it would only ever be called on initial mount and then on a re-mount so changing things within the system would not recall the api.

Reply:
Yeah that makes sense